homeassistant/helpers/frame.py

```diff
@@ -129,15 +129,19 @@ class MissingIntegrationFrame(HomeAssistantError):
 
 def report(
     what: str,
+    *,
     exclude_integrations: set | None = None,
```

Comment:
Could/Should this be a set of strings?
```suggestion
    exclude_integrations: set[str] | None = None,
```